include/vsg/maths/color.h

```diff
@@ -99,28 +99,28 @@ namespace vsg
     template<typename T>
     constexpr t_vec4<T> linear_to_sRGB(const t_vec4<T>& src)
     {
-        const T exponent = static_cast<T>(2.2);
+        const T exponent = static_cast<T>(1.0 / 2.2);
```

Comment:
What testing did you use to determine that the conversion was wrong?  You've flipped vsg::Window constructor and this local code so that Window.cpp code is now doing exactly the same as before but with flipped meaning.

Reply:
I was feeding `sRGB_to_linear` sRGB colours with the vsgXchange PR, and getting nonsensical results. When switched, I got correct results. It's pretty easy to find sources saying these functions should work the way this PR makes them work. For a start, the conversion functions in the standard VSG PBR shader work this way.

Reply:
I noticed that shaders used the ratios in your PR.

The change you've made to Window.cpp still seem odd.  If the frame buffer is sRGB when would you convert to sRGB to linear?

Reply:
I've left a comment on that thread about it. When the framebuffer has an sRGB format, it means Vulkan expects to be fed linear colour and will then do a to-sRGB conversion any time anything is written to the framebuffer.

Reply:
Um.... OK, so that kinda explains the perverse conversion in Window.cpp on the _clearColor, but it's not a sRGB_to_linear conversion, it's really a preconversion of a linear colour to something that remains the approrpiate colour when the framebuffer operation converts the linear colour to sRGB.  Mathematically I can see that it's the same as sRGB_to_linear but it's not actually doing that conversion.